packages/network-of-terms-catalog/catalog/datasets/muziekschatten-onderwerpen.jsonld

```diff
@@ -22,7 +22,7 @@
     }
   ],
   "url": [
-    "https://data.muziekschatten.nl/som/"
+    "https://data.muziekschatten.nl/"
```

Comment:
Red alert: does this mean URIs have changed? Or just that _new_ concepts may not have `/som/` in their URL?